### ISSUE TITLE
litecoin support

### ISSUE THREAD

**Issue by seb-sykio** [GemHQ/money-tree#53]
hi

I think this should work for litecoin
`
        litecoin: {
          address_version: '30',
          p2sh_version: '32',
          p2sh_char: '3',##TODO
          privkey_version: 'b0',
          privkey_compression_flag: '01',##TODO
          extended_privkey_version: "019d9cfe",
          extended_pubkey_version: "019da462",
          compressed_wif_chars: %w(K L),##TODO
          uncompressed_wif_chars: %w(5),##TODO
          protocol_version: 70002
        },
        litecoin_testnet: {
          address_version: '6f',
          p2sh_version: '3a',
          p2sh_char: '2',##TODO
          privkey_version: 'ef',
          privkey_compression_flag: '01',##TODO
          extended_privkey_version: "0436ef7d",
          extended_pubkey_version: "0436f6e1",
          compressed_wif_chars: %w(c),##TODO
          uncompressed_wif_chars: %w(9),##TODO
          protocol_version: 70002
        }`
could you add it to NETWORK ?
thanks

**Comment by q9f** [comment]
Sure. It's already done in #34 

I'll review this asap.

**Comment by krtschmr** [comment]
Yes, this would work. I haven't seen this PR and mentions as i was unlawfully arrested and spent 8 months in soltitary confinement. I'll see if i can have some time on the weekend and fix merge conflicts and write a spec.
